src/Player.js

```diff
@@ -13,6 +18,13 @@ export default class Player {
     this.shot = {x1: 0, x2: 100, y1: 0, y2: 100, alpha: 0};
     this.hasShot = true;
 
+    if (socket) {
+      socket.on('startingPos', ({x,y}) => {
+        this.x = x * blockWidth;
+        this.y = y * blockHeight;
+      });
+    }
+
```

Comment:
in theory if they move before they spawn it might be possible to start in the 0-0 position

Reply:
0-0 should be fine if it's not a wall block.